proposals/experimental.md

```diff
@@ -34,100 +36,113 @@ package kotlin
+In a previous version of this proposal, we discussed the possibility of introducing another argument, `-Xexperimental=org.foo.Ann`, to use the propagating opt-in on the whole module (i.e. mark the whole module as "experimental" in terms of that proposal). The implementation of that feature turned out to be unexpectedly complicated, and it wasn't widely used, so we've decided not to add it at this point.
 
-## Experimentality of Experimental/UseExperimental itself
+## Opt-in requirement of RequiresOptIn/OptIn themselves
 
-Annotations `Experimental` and `UseExperimental` are proposed to be added to the standard library in Kotlin 1.3. Since we're not yet sure that this design is optimal, we would like to test it first, and see if we can finalize it. Therefore, we would like to keep this whole feature experimental itself, in the sense that we may change something incompatibly, and the client code must be aware of it.
+Annotations `RequiresOptIn` and `OptIn` are proposed to be added to the Kotlin standard library. Since we're not yet sure that this design is optimal, we would like to test it first, and see if we can finalize it. Therefore, we would like to keep this whole feature experimental itself, in the sense that we may change something incompatibly, and the client code must be aware of it.
 
-Therefore, we will **require** each user of `Experimental` to provide at least one unstable (`-X...`) compiler argument, which would mean that the user is understanding the risks of using the experimental functionality. It can be either `-Xexperimental=...` or `-Xuse-experimental=...` with a specific marker, or the magic predefined argument `-Xuse-experimental=kotlin.Experimental` which doesn't allow using any experimental API by itself, yet merely allows using `Experimental` and `UseExperimental` in the source code. Unless one of these arguments is provided, the compiler will report a warning on each usage of `Experimental` or `UseExperimental` (but not on usages of the markers!).
+Therefore, we will **require** each user of `RequiresOptIn` to provide at least one unstable (`-X...`) compiler argument, which would mean that the user is understanding the risks of using this experimental functionality. It can be either `-Xopt-in=...` with any opt-in requirement marker, or the magic predefined argument `-Xopt-in=kotlin.RequiresOptIn` which doesn't allow using any API by itself, yet merely allows using `RequiresOptIn` and `OptIn` in the source code. Unless one of these arguments is provided, the compiler will report a warning on each usage of `RequiresOptIn` or `OptIn` (but not on usages of the markers!).
```

Comment:
> to provide at least one unstable (`-X...`) compiler argument

Is any `-X...` argument enough, e.g. `-Xinline-classes`?

Reply:
No, it was poorly worded, fixed.